factory/outportBlockCasterFactory_test.go

```diff
@@ -0,0 +1,47 @@
+package factory
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/multiversx/mx-chain-core-go/data/outport"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetOutportCaster(t *testing.T) {
+	t.Run("get heaverV1 caster", func(t *testing.T) {
+
```

Comment:
```suggestion
func TestGetOutportCaster(t *testing.T) {
        t.Parallel()
        
	t.Run("get heaverV1 caster", func(t *testing.T) {
	         t.Parallel()

```
please use `t.Parallel()` in all tests and inner tests

Reply:
added `t.Parallel()` but in the only test that survived the refactoring.